hash/apple2_flop_clcracked.xml

```diff
@@ -3000,4 +3005,3800 @@
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="alnaddv2">
+		<description>Alien Addition revision 2 (cleanly cracked)</description>
+		<year>1982</year>
+		<publisher>Developmental Learning</publisher>
+		<info name="release" value="2015-11-07"/>
+		<!-- No compatibility data known at this time. -->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="alien addition revision 2 (4am crack).dsk" size="143360" crc="431ff422" sha1="3b31f7f70b0b36a291fb32cf146ceb300be676a9" offset="0x0000" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="alnadd82">
+		<description>Alien Addition 1982.0 (cleanly cracked)</description>
+		<year>1982</year>
+		<publisher>Developmental Learning</publisher>
+		<info name="release" value="2017-09-08"/>
+		<!-- No compatibility data known at this time. -->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="alien addition 1982.0 (4am crack).dsk" size="143360" crc="ab9ad63c" sha1="3af54aab6b6001bcb5e40a6fbeee4540e44f938c" offset="0x0000" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="alnadd85">
+		<description>Alien Addition 1985 (cleanly cracked)</description>
+		<year>1985</year>
+		<publisher>Developmental Learning</publisher>
+		<info name="release" value="2017-06-29"/>
+		<!-- No compatibility data known at this time. -->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="alien addition 1985 (4am crack).dsk" size="143360" crc="6f49910e" sha1="1e5bceb1c965f06d11af6fb8f21bd2bbc9564665" offset="0x0000" />
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
+	<!-- **************************************************************************************************************** -->
```

Comment:
What's this big separator for?

Reply:
I left that in? Sorry, that was demarcating a point where I was working across two files and needed to be particularly careful with what was below that point. It'll be gone with the next PR I give you guys.